### ISSUE TITLE
Add block functionality in addition to shortcode

### ISSUE THREAD

**Issue by lbell** [lbell/pretty-google-calendar#16]
Maybe...

**Comment by mwguerra** [comment]
Hi @lbell,

Considering the Gutenberg block is a good idea, but abandoning shortcodes might pose a challenge for some developers who use other page builders like Divi or Elementor. There seems to be a significant shift towards adapting shortcodes to blocks, but I believe maintaining the availability of shortcodes is also important.

**Comment by lbell** [comment]
Good point! It's just that the shortcode parameters are getting unwieldy, but I see what you mean. 

No reason we can't do both...

**Comment by lbell** [comment]
Added in https://github.com/lbell/pretty-calendar-embeds!
